t04_refactored.py

```diff
@@ -533,14 +536,48 @@ def team_19_adv():
         print("Yummy! But not few and far between in these lands.")
     kill_if_dead(dead)
 
+
 def team_20_adv():
     pass
     # TODO Add your code here
 
 
 def team_21_adv():
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/16WddE8AhpLg8duTmMrZPy8rBFTPxWAWdfpYzfPK1m_w/edit?usp=sharing
+    Cole Collins
+    """
+    global dead
+
+    print("\n")
+    print("You find a room that contains two gold statues.")
+    sleep(delay)
+    statues = input("Do you take the left or right statue, or don't pick up the statue?: ")
+    print("\n")
+
+    if statues == "left":
+        # good choice
+        print("You pick up the statue on the left.")
+        sleep(delay)
+        print("A passageway opens and you see outside.")
+        print("Congrats! You escaped the cave!")
+        sleep(delay)
+    elif statues == "right":
+        # bad choice
+        print("You pick up the statue on the right.")
+        sleep(delay)
+        print("A passageway opens above you.")
+        sleep(delay)
+        print("A bunch of snakes and spiders fall on you!")
+        dead = True
+        sleep(delay)
+    else:
+        # neutral choice
+        print("You don't pick up either of the statues.")
+        sleep(delay)
+        print("Nothing happens... You are still stuck in the cave.")
+        sleep(delay)
+kill_if_dead(dead)
```

Comment:
This is outside of the function. Indent it one (or two) lines to put it inside the function and/or inside the `else` statement.